app/views/pipeline_jobs/index.html.erb

```diff
@@ -90,7 +90,13 @@
                     <%= report.status.capitalize %>
                   </span>
                 </td>
-                <td><%= pipeline_job.start_time.strftime('%H:%M %d/%m/%y') %></td>
+
+                <% if report.harvest_job&.extraction_job.present? %>
```

Comment:
My only concern with this I think is that if you run a harvest that is using an existing extraction it would show the time that that extraction first ran. Can you check what happens in this scenario?